CHANGELOG.rst

```diff
@@ -18,32 +18,47 @@ number of the code change for that issue.  These PRs can be viewed at:
     https://github.com/spacetelescope/drizzlepac/pulls
 
 
-3.6.3 (unreleased)
-=====================
+3.7.0rc1 (21-Feb-2024) Infrastructure Build
+===========================================
+
+- Update project.toml file to specify numpy>=1.18,  <2.0 [#1743]
+
+- Update project.toml file to specify python_requires>=3.10 [#1737]
+
 - Github branch "master" renamed to main. [#1725]
 
-- Reorganized the readthedocs documentation with the help of various STScI staff. [#1717] 
+- Clean up spacing in toml file to eliminate improper spacing to
+  avoid decprecation warning [#1731]
+
+- Clean up YAML diagram in of workflows area [#1728]
+
+- Updated installation instructions and small text changes [#1727]
 
-- Added FluxAp1, FluxErrAp1, FluxAp2, and FluxErrAp2 to point sources catalogs 
-  as a part of a bug fix for negative magnitude errors and consequently false 
-  positives in the flagging. [#1700]
+- Remove outdated references of Pyraf and change to Python [#1726]
 
-- Updated the Segmentation catalog so the evaluation of the segmentation image using
-  the RickerWavelet kernel always allows for crowded fields (check_big_island_only = True)
-  because a large fraction of the image will covered by source segments. Changed the ratio
-  of Gaussian vs RickerWavelet "big source" evaluation to use Round 2 computations, and
-  added more log messages for clarification. [#1730]
+- Fix to add "stregion" to the requirements-dev.txt file to fix the build
+  error under Python 3.12. [#1714]
 
+- Reorganized the readthedocs documentation with the help of various STScI
+  staff. [#1717] 
+
+- Updates requirements-dev.txt to not install eggs which causes problems
+  for the regression tests [#1721]
```

Comment:
This sentence can be read in two way. Maybe:
Updates requirements-dev.txt to not install eggs that cause problems or the regression tests

Reply:
Updated.